src/api/hello.ts

```diff
@@ -0,0 +1,13 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+type Data = {
+  name: string
+}
+
+export default function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  res.status(200).json({ name: 'John Doe' })
+}
```

Comment:
I tried the `localhost:3000/api/hello`, but I could not get this to work. We will need the next-js-api, so it would be nice that this is initialized and tested that it works!

Reply:
or not sure if the api-route is used, but the static server pages at least :)

Reply:
i ll check

Reply:
fixed